packages/frontend/src/ee/features/embed/EmbedDashboard/hooks.ts

```diff
@@ -10,21 +10,17 @@ import useDashboardFiltersForTile from '../../../../hooks/dashboard/useDashboard
 import useDashboardContext from '../../../../providers/Dashboard/useDashboardContext';
 import { postEmbedChartAndResults, postEmbedDashboard } from './api';
 
-export const useEmbedDashboard = (
-    projectUuid: string | undefined,
-    embedToken: string | undefined,
-) => {
+export const useEmbedDashboard = (projectUuid: string | undefined) => {
     return useQuery<Dashboard & InteractivityOptions, ApiError>({
         queryKey: ['embed-dashboard'],
-        queryFn: () => postEmbedDashboard(projectUuid!, embedToken!),
-        enabled: !!embedToken && !!projectUuid,
+        queryFn: () => postEmbedDashboard(projectUuid!),
+        enabled: !!projectUuid,
```

Comment:
will these changes be compatible with existing saved embedded dashboards tokens ?

Reply:
That's a good question. My assumption was that we wouldn't even make it here because of early returns in the components using these hooks. I can add them back to be safe.